src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1951,7 +2062,7 @@ void window_guest_thoughts_resize(rct_window* w)
         window_invalidate(w);
     }
 
-    window_set_resize(w, 192, 159, 500, 450);
+    window_set_resize(w, 192 + (gConfigGeneral.debugging_tools ? TabWidth : 0), 159, 500, 450);
```

Comment:
You may as well make a separate function call for this to save the 7 repetitions of it.